@codex please fix

```
=========================== short test summary info ============================
FAILED tests/test_tailscale_remote_ops_e2e.py::test_tailscale_recipes_e2e_with_stubs - AssertionError: scripts/tailscale_remote_ops.sh install
  missing or invalid SUGARKUBE_TAILSCALE_INSTALL_SHA256 (expected 64 hex chars).
  Refusing to execute a downloaded installer script without integrity verification.
  error: Recipe `tailscale-install` failed on line 199 with exit code 1
  
assert 1 == 0
 +  where 1 = CompletedProcess(args=['/usr/bin/just', 'tailscale-install'], returncode=1, stdout='', stderr='scripts/tailscale_remote_ops.sh install\nmissing or invalid SUGARKUBE_TAILSCALE_INSTALL_SHA256 (expected 64 hex chars).\nRefusing to execute a downloaded installer script without integrity verification.\nerror: Recipe `tailscale-install` failed on line 199 with exit code 1\n').returncode
====== 1 failed, 1156 passed, 21 skipped, 9 warnings in 151.75s (0:02:31) ======
Error: Process completed with exit code 1.
```

new codex task, not a r/e/v/i/e/w task

--- a/tests/test_tailscale_remote_ops_e2e.py
+++ b/tests/test_tailscale_remote_ops_e2e.py
@@ -72,6 +72,9 @@ printf 'tailscale %s\n' "$*" >> "{calls}"
     env = os.environ.copy()
     env["PATH"] = f"{fakebin}:{env.get('PATH', '')}"
     env["SUGARKUBE_TAILSCALE_AUTH_KEY"] = "tskey-test"
+    env["SUGARKUBE_TAILSCALE_INSTALL_SHA256"] = (
+        "fb99eae951f1adc14d1a4a9a186c21930db2786b3208c94c7d9af382bd1048e5"
+    )
 
     install = subprocess.run(
         [just_bin, "tailscale-install"],

